moto/__init__.py

```diff
@@ -171,22 +172,24 @@ def f(*args, **kwargs):
 mock_sdb = lazy_load(".sdb", "mock_sdb", boto3_name="sdb")
 
 
-def mock_all():
-    dec_names = [
-        d
-        for d in dir(sys.modules["moto"])
-        if d.startswith("mock_")
-        and not d.endswith("_deprecated")
-        and not d == "mock_all"
-    ]
+class mock_all(ContextDecorator):
```

Comment:
nitpick from my side as well 😶‍🌫️ 
WDYT about:
```python 
class MockAll(ContextDecorator):
    ...

#in the end of the file:
mock_all = MockAll()
```

Reply:
Good catch - that is a better (more pythonic) naming scheme :+1: